projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts

```diff
@@ -207,9 +207,11 @@ export class EntityTableDataSourceModel extends TableDataSourceModel {
       case FilterOperator.Equals:
         return rows.filter(eachRow => ((eachRow[actualKeyNameInRows] as Entity).name as string) === tableFilter.value);
       case FilterOperator.Like:
-        const regexExp = new RegExp(tableFilter.value as string);
+        const regexExp = new RegExp((tableFilter.value as string).toLowerCase());
```

Comment:
How about adding the [ignoreCase][1] flag instead?

```suggestion
        const regexExp = new RegExp((tableFilter.value as string), "i");
```

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/ignoreCase

Reply:
Testing this out

Reply:
Thanks foe the suggestion, updated the regex to use the flag.